Gemfile

```diff
@@ -10,3 +10,5 @@ gem 'sqlite3'
 
 # Specify your gem's dependencies in core_data_connector.gemspec.
 gemspec
+
+gem "rexml", "~> 3.2"
```

Comment:
Out of curiosity, where are we using this library?

Reply:
`Hash.from_xml(body).to_json` would error out in the console, and the error message said to install `rexml`. It worked after installing that gem.

Reply:
See https://github.com/rails/rails/blob/c7551d08fc572655105da38394d8672b8259c22f/activesupport/lib/active_support/xml_mini/rexml.rb#L41

Reply:
Ahh, makes sense. I think we can just add the dependency to the `gemspec` file. The only reason we have dependencies in here are to support gems loaded from GitHub (which is a bundler feature).